tests/ui/check-cfg/well-known-values.stderr

```diff
@@ -156,7 +156,7 @@ warning: unexpected `cfg` condition value: `_UNEXPECTED_VALUE`
 LL |     target_env = "_UNEXPECTED_VALUE",
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
-   = note: expected values for `target_env` are: ``, `gnu`, `msvc`, `musl`, `newlib`, `nto70`, `nto71`, `ohos`, `p1`, `p2`, `relibc`, `sgx`, and `uclibc`
+   = note: expected values for `target_env` are: ``, `gnu`, `msvc`, `msys2`, `musl`, `newlib`, `nto70`, `nto71`, `ohos`, `p1`, `p2`, `relibc`, `sgx`, and `uclibc`
```

Comment:
For example, accepting `target_env = ""` for cygwin will make `cfg(target_env = "")` become a legal value (a well-known value, even), which I suspect is never what we want.

Reply:
Well, yes. But it's already a valid value without this PR, I think...

Reply:
Hm, you're right, nevermind.